CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
 
+## [1.51.2](https://github.com/brave/brave-ios/releases/tag/v1.51.2)
+- Fixed ads event reporting for sponsored images on new tab page. ([#7566](https://github.com/brave/brave-ios/issues/7566))
```

Comment:
nit re: formatting: Should be a space between the release version and the release notes. We also usually capitalize New Tab Page.